instrumentation/jmx-metrics/library/src/main/java/io/opentelemetry/instrumentation/jmx/yaml/RuleParser.java

```diff
@@ -51,17 +51,18 @@ public JmxConfig loadConfig(InputStream is) throws Exception {
    *
    * @param conf the metric configuration
    * @param is the InputStream with the YAML rules
+   * @param id identifier of the YAML ruleset, such as a filename
    */
-  public void addMetricDefsTo(MetricConfiguration conf, InputStream is) {
+  public void addMetricDefsTo(MetricConfiguration conf, InputStream is, String id) {
     try {
 
       JmxConfig config = loadConfig(is);
       if (config != null) {
-        logger.log(INFO, "Found {0} metric rules", config.getRules().size());
+        logger.log(INFO, id + ": found {0} metric rules", config.getRules().size());
```

Comment:
Can you use placeholders?
```suggestion
        logger.log(INFO, "{0}: found {1} metric rules", new Object[] {id, config.getRules().size()});
```